### PR TITLE
Add remove app procedure

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9083

### CODE REVIEW

File: Teams/upload-custom-apps.md [hunk 1]
```diff
@@ -114,6 +114,14 @@ You can update the app on the Manage apps page in the Microsoft Teams admin cent
 
 In most cases, after you complete an app update the new version automatically appears for end users. For more information, see [end-user update experience](apps-update-experience.md).
 
+## Remove
+
+To remove an app:
+
+1. In the left navigation of the Teams admin center, go to **Teams apps** > **Manage apps**.
+2. Click the app name to go to the app details page, and then select the three dots **...** > **Actions** > **Delete**.
```
Comment (JohanFreelancer9) [inline Teams/upload-custom-apps.md:122]:
```suggestion
2. Select the app name to go to the app details page, and then select the three dots **...** > **Actions** > **Delete**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ashishguptaiitb PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ashishguptaiitb** [comment]
I am updating the doc via another internal-only PR ( https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr/pull/9940/commits/f8aaa158d232109c5d99fffb6bd292844cb78695 ) and it'll go live in a few hours. I suggest that we close this PR.

**Comment by get-itips** [comment]
@ashishguptaiitb please notify when your PR goes live so I can close the issue

**Pull request closed without merging**